content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -176,9 +179,16 @@ network of the consumers.
 ## Control plane models
 
 An Istio mesh uses the {{< gloss >}}control plane{{< /gloss >}} to configure all
-communication between workload instances within the mesh. You can replicate the
-control plane, and workload instances connect to any control plane instance to
-get their configuration.
+communication between workload instances within the mesh. At one extreme, you can choose
+to deploy one control plane per cluster in the mesh, while at the other extreme,
+you can host one single shared control plane for all clusters in the mesh.
+
+_Note: Each control plane will only honor configurations (such as Gateways, VirtualServices, etc.)
+that are present in its local Kubernetes cluster. If you wish to apply a configuration to all workloads in the mesh
+across control planes, it is your responsibility to replicate the configuration to the
+Kubernetes clusters associated with each control plane._
```

Comment:
Think this `it is your responsibility to replicate the configuration` - do you mean istio config or k8s objects for services?  I assume it is later for shared control plane and perhaps both for replicated control plane... this can generate confusion to users.